### PR TITLE
ci: add chromatic support

### PR DESCRIPTION
**Pull request by matthprost**
## Type

-  CI

#### What is expected?

Added chromatic to the project.

### CODE REVIEW

**Review by DorianMaliszewski — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -106,3 +106,18 @@ jobs:
...
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3.1.0
+        with:
+          fetch-depth: 0
+      - uses: pnpm/action-setup@v2.2.3
+      - name: Install dependencies
+        run: pnpm install
+      - name: Publish to Chromatic
+        if: github.ref != 'refs/heads/main'
```
Comment (chambo-e) [inline .github/workflows/ci.yml:119]: why not on main ?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -106,3 +106,18 @@ jobs:
...
+        uses: chromaui/action@v1
```
Comment (chambo-e) [inline .github/workflows/ci.yml:120]: Can we retrieve storybookUrl outputted by this action to deploy a new GitHub environment (thus removing our deploy) ? https://github.com/chromaui/action/blob/b9b365f0c7d27357f2f73211bd3c9a58691d2aaf/action.yml#L119

File: .github/workflows/ci.yml (hunk 1 above)
Comment (matthprost) [reply .github/workflows/ci.yml:119]: @chambo-e My bad I tough we will still stay on our infa for deploy

File: .github/workflows/ci.yml (hunk 2 above)
Comment (matthprost) [reply .github/workflows/ci.yml:138]: @chambo-e should be better 😄

**Review by chambo-e — COMMENTED** [review]
Waiting for the bot activation to see how it's handled :)

**Review by baptisteArno — APPROVED** [review]

**Review by chambo-e — APPROVED** [review]

**Pull request merged**